File: birthday_cake_candles/purplepixie/BirthdayCake.py
#!/bin/python3

import sys

def birthdayCakeCandles(n, ar):
    highest = max(ar)
    count = ar.count(highest)
    return count

n = int(input().strip())
ar = list(map(int, input().strip().split(' ')))
result = birthdayCakeCandles(n, ar)
print(result)

File: birthday_cake_candles/purplepixie/BirthdayCake.rb
#!/bin/ruby

def birthdayCakeCandles(n, ar)
    max = ar.max
    count = ar.count(max)
    return count
end

n = gets.strip.to_i
ar = gets.strip
ar = ar.split(' ').map(&:to_i)
result = birthdayCakeCandles(n, ar)
puts result;
